internal/kubernetes/kubernetes.go

```diff
@@ -630,7 +624,7 @@ func (k *Kubernetes) Endpoint() (string, int, error) {
 		return k.NodePort(service)
 	case corev1.ServiceTypeLoadBalancer:
 		if len(service.Status.LoadBalancer.Ingress) == 0 {
-			return "", -1, fmt.Errorf("load balancer has no ingress objects")
+			return "", -1, fmt.Errorf("no address available for load balancer")
```

Comment:
Maybe something to indicate it may still resolve itself? Ex:

> no address available for load balancer yet, it may still be provisioning

Or something along those lines